utils/cups-browsed.c

```diff
@@ -153,6 +153,8 @@ static int  ldap_rebind_proc(LDAP *RebindLDAPHandle,
 #define REMOTE_DEFAULT_PRINTER_FILE "/cups-browsed-remote-default-printer"
 #define SAVE_OPTIONS_FILE "/cups-browsed-options-%s"
 #define DEBUG_LOG_FILE "/cups-browsed_log"
+#define DEBUG_LOG_FILE_2 "/cups-browsed_previous_logs"
+#define MAX_LOG_SIZE 30
```

Comment:
Should be configurable in cups-browsed.conf.
Which unit is this "30"? kBytes? MBytes?

Reply:
See commit b6cbd1fec as an example for adding an option to cups-browsed.conf.

Reply:
The unit is that of KBytes

Reply:
In https://github.com/OpenPrinting/cups-filters/pull/267/commits/d8e30a81bac7bcdfc8d1399bf204328d36ae866e you forgot to read out the setting from `cups-browsed.conf`. All the rest (defining and using the global variable, man page, cups-browsed.conf comments) you did correctly.

Reply:
Also nice to allow the value "0" to turn off this facility and log into one file as before, when implementing that do not forget to mention in the man page and the cups-browsed.conf file.